packages/shared/src/write-archive/index.ts

```diff
@@ -38,9 +38,7 @@ export async function writeTestResult(
   );
   // in Storybook, `/` splits the title out into hierarchies (folders)
   const title = titlePathWithoutFileExtensions.join('/');
-  // outputDir gives us the test-specific subfolder (https://playwright.dev/docs/api/class-testconfig#test-config-output-dir);
-  // we want to write one level above that
-  const finalOutputDir = join(outputDir, '..', 'chromatic-archives');
+  const finalOutputDir = join(outputDir, 'chromatic-archives');
```

Comment:
@tevanoff it seems like the directory traversal wouldn't be as much of a risk since people are configuring their own `outputDir`, which looks on "their" OS/container for files. Is it worth making this change to be totally safe, or is `outputDir` a relative path?